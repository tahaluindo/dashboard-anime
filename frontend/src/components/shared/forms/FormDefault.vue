<template>
  <form class="grid gap-4" autocomplete="off" data-netlify="true">
    <slot />
  </form>
</template>
