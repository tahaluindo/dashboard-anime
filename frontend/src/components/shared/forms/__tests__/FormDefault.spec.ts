import { mount } from "@vue/test-utils";
import FormDefault from "../FormDefault.vue";

describe("FormDefault component test", () => {
  it("should render the default component.", () => {
    const wrapper = mount(FormDefault, {
      slots: {
        default: "<div> TEST </div>",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
