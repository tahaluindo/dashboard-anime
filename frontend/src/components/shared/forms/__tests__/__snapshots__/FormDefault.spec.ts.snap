// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormDefault component test should render the default component. 1`] = `
"<form class=\\"grid gap-4\\" autocomplete=\\"off\\" data-netlify=\\"true\\">
  <div> TEST </div>
</form>"
`;
